security/bug-bounty.md

```diff
@@ -1,32 +1,61 @@
-Clever is committed to working with security experts across the world to stay up to date with the latest security techniques. If you have discovered a security issue that you believe we should know about, we'd welcome working with you. You can read about our security and privacy efforts on our [security page](https://clever.com/security)
-
-**Coordinated disclosure rules**
+Clever is committed to working with security experts around the world to stay up to date with the latest security techniques. If you have discovered a security issue that you believe we should know about, we'd love to hear from you. You can read about our security and privacy efforts on our [security page](https://clever.com/security).
 
+Coordinated disclosure rules
+----------------------------
 * Please let us know as soon as possible upon discovery of a potential security issue, and we’ll make every effort to quickly correct the issue.
-* Provide us a reasonable amount of time to fix the issue before publishing it elsewhere.
-* Making a good faith effort to not leak, manipulate, or destroy any user data. Please only test against accounts you own yourself or with explicit permission of the account holder.
+* Provide us with a reasonable amount of time to fix the issue before publishing it elsewhere.
+* Make a good faith effort to not leak, manipulate, or destroy any user data, or affect the availability of the site. Please only test against accounts you own yourself or with explicit permission of the account holder.
```

Comment:
Can we add this to eligibility also? No user data access, account takeover should be done